addon/components/frost-sort-item.js

```diff
@@ -7,11 +7,13 @@ const {
 import PropTypesMixin, { PropTypes } from 'ember-prop-types'
 import computed from 'ember-computed-decorators'
 import layout from '../templates/components/frost-sort-item'
+import uuid from 'ember-simple-uuid'
 
 export default Component.extend(PropTypesMixin, {
   // == Properties ============================================================
   layout: layout,
   classNames: ['frost-sort-item'],
+  targetOutlet: `frost-sort-${uuid()}`,
```

Comment:
this is internal to the sort component itself. so I don't think we want any customisation from the user side.